exonum/src/storage/values.rs

```diff
@@ -240,3 +241,72 @@ impl StorageValue for String {
         String::from_utf8(value.into_owned()).unwrap()
     }
 }
+
+/// Uses little-endian encoding.
+impl CryptoHash for SystemTime {
```

Comment:
It seems other `CryptoHash` implementations are in `crypto` module. Is there a particular reason this one is here?